hadoop-common-project/hadoop-common/src/main/native/src/org/apache/hadoop/io/compress/snappy/SnappyCompressor.c

```diff
@@ -1,166 +0,0 @@
-/*
```

Comment:
Per https://github.com/apache/hadoop/pull/2201#issuecomment-681687572 Are those native code used in `hadoop-mapreduce-client-nativetask`? If so, we probably need to keep it now.

Reply:
Hmm, because we remove native method in java files, I think we don't generate .h file needed for compilation: https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2297/1/artifact/out/patch-unit-hadoop-common-project_hadoop-common.txt

```
[WARNING] /home/jenkins/jenkins-home/workspace/hadoop-multibranch_PR-2297/src/hadoop-common-project/hadoop-common/src/main/native/src/org/apache/hadoop/io/compress/snappy/SnappyDecompressor.c:32:10: fatal error: org_apache_hadoop_io_compress_snappy_SnappyDecompressor.h: No such file or directory
[WARNING]  #include "org_apache_hadoop_io_compress_snappy_SnappyDecompressor.h"
[WARNING]           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[WARNING] compilation terminated.
```

Reply:
Btw, I don't see they are used in `hadoop-mapreduce-client-nativetask` if I don't miss it. Let's wait the build and test.